### PR TITLE
[PBE-163] SDK update, python 3.11, django 4.1.0

### PR DESCRIPTION
**Pull request by itsmeadi**

### CODE REVIEW

File: stream_django/tests/test_app/migrations/0001_initial.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 
 
 class Migration(migrations.Migration):
-
```
Comment (itsmeadi) [inline stream_django/tests/test_app/migrations/0001_initial.py:11]: lint fixes

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -11,8 +11,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: [3.6, 3.7, 3.8, 3.9, "3.10"]
```
Comment (itsmeadi) [inline .github/workflows/ci.yml:14]: removed python 3.6, not supported
https://github.com/actions/setup-python/issues/544#issuecomment-1332535877

File: setup.py
Comment (JimmyPettersson85) [inline setup.py:1]: We can remove `2.7` and `3.6` from this file.

Comment (itsmeadi) [reply setup.py:1]: removed

**Review by JimmyPettersson85 — APPROVED** [review]
LGTM

**Review by hdiogenes — APPROVED** [review]

**Pull request merged**